inspire_dojson/hep/model.py

```diff
@@ -90,6 +92,18 @@ def copy_special_collections(record, blob):
     return record
 
 
+def move_journal_letters(record, blob):
+    for publication_info in get_value(record, 'references.reference.publication_info', default=[]):
+        journal_title = publication_info.get('journal_title')
+        journal_volume = publication_info.get('journal_volume')
+        if journal_title and journal_volume:
+            if journal_volume[0] in string.ascii_uppercase:
+                publication_info['journal_title'] += journal_volume[0]
+                publication_info['journal_volume'] = journal_volume[1:]
+
+    return record
```

Comment:
Wait, this isn't right! In fact, we probably want to add this to the `ReferenceBuilder`, otherwise we'd have to fix this everywhere...